src/librustc_metadata/cstore_impl.rs

```diff
@@ -129,6 +132,29 @@ provide! { <'tcx> tcx, def_id, cdata
         cdata.const_is_rvalue_promotable_to_static(def_id.index)
     }
     is_mir_available => { cdata.is_item_mir_available(def_id.index) }
+
+    dylib_dependency_formats => { Rc::new(cdata.get_dylib_dependency_formats(&tcx.dep_graph)) }
+    is_allocator => { cdata.is_allocator(&tcx.dep_graph) }
+    is_panic_runtime => { cdata.is_panic_runtime(&tcx.dep_graph) }
+    extern_crate => { Rc::new(cdata.extern_crate.get()) }
+}
+
+pub fn provide_local<'tcx>(providers: &mut Providers<'tcx>) {
```

Comment:
> which `rustc_metadata` is not about

Why did it used to be [a method on `CStore`](https://github.com/rust-lang/rust/pull/42598/files#diff-f682599ab8df719f4fa313038a5d2f72L175)?